frontend/src/employee-mobile-frontend/components/attendances/actions/MarkAbsentBeforehand.tsx

```diff
@@ -74,6 +86,12 @@ export default React.memo(function MarkAbsentBeforehand() {
     loadFutureAbsences(childId)
   }, [loadFutureAbsences, childId])
 
+  useEffect(() => {
+    if (absences.isSuccess && absences.value.length > 1) {
+      setAbsenceRanges(groupAbsencesByDateRange(absences))
```

Comment:
Do we need the original absences? We could pass a set function into useRestApi which already does the mapping.